server/models/Update.js

```diff
@@ -121,6 +121,16 @@ export default function(Sequelize, DataTypes) {
 
       image: DataTypes.STRING,
 
+      isPrivate: {
+        type: DataTypes.BOOLEAN,
+        defaultValue: false,
+      },
+
+      contentStripped: {
```

Comment:
Maybe prefix with `is`. I'm wondering about a better name but I'm not decided yet, ie: `isContentAvailable`. Let's see what @Betree thinks about that.

Reply:
Yes, `isContentAvailable` is a better name.

Reply:
I've renamed it to `isContentAvailable`.